### ISSUE TITLE
Perl-5.30.0-GCCcore-8.3.0.eb fails sanity check due to "no (non-empty) directory found at 'man'"

### ISSUE THREAD

**Issue by terjekv** [EESSI/software-layer#18]
This builds fine without `--sysroot`, but with `--sysroot` we get the following:
```
= FAILED: Installation ended unsuccessfully (build directory: /tmp/terjekv/easybuild/build/Perl/5.30.0/GCCcore-8.3.0): build failed (first 300 chars): Sanity check failed: no (non-empty) directory found at 'man' in /tmp/terjekv_eessi/software/Perl/5.30.0-GCCcore-8.3.0 (took 38 min 13 sec)
== Results of the build can be found in the log file(s) /tmp/eb-b3202lrw/easybuild-Perl-5.30.0-20200920.202007.hhRkA.log
ERROR: Build of /cvmfs/pilot.eessi-hpc.org/2020.08/software/x86_64/intel/haswell/software/EasyBuild/4.3.0/easybuild/easyconfigs/p/Perl/Perl-5.30.0-GCCcore-8.3.0.eb failed (err: "build failed (first 300 chars): Sanity check failed: no (non-empty) directory found at 'man' in /tmp/terjekv_eessi/software/Perl/5.30.0-GCCcore-8.3.0")
```
I seem to recall perl being fixed for sysroot, but I can't see why 5.30.2-GCCcore-9.3.0 works and this doesn't.

**Comment by boegel** [comment]
I built `Perl` with an extra build dependency in the 2020.08 version of the pilot repo, see https://github.com/easybuilders/easybuild-easyconfigs/pull/11200 .

That PR hasn't been merged yet because `groff` requires `texinfo` as a build dep (for the `makeinfo` command), which requires `texlive` to be installed as an OS dependency...

Using `--ignore-osdeps` it turns out that `texinfo` installs just fine in a Gentoo Prefix environment however, so it seems the `texlive` is only a runtime dependency.

We want to check if we can put together a trimmed down version of `texinfo` that only provides `makeinfo`, which is the only part that is actually required as a build dependency for `groff`...

**Comment by terjekv** [comment]
Dependency hell has made us reach TexLive. Golly me, be still my heart. ./sigh.

**Comment by bedroge** [comment]
We can always add texlive to the compatibility layer, if necessary. Should be quite trivial.

**Comment by boegel** [comment]
That won't fix anything, the problem is the missing `groff` build dep for `Perl`. And I would really like to avoid having to install `texlive` (a full LaTeX installation...) to the compatibility later just to install `Perl`.

**Comment by boegel** [comment]
fixed in 2020.09 pilot stack (via https://github.com/easybuilders/easybuild-easyconfigs/pull/11200)
